includes/class-reader-activation.php

```diff
@@ -737,10 +737,11 @@ public static function set_current_reader( $user_id ) {
 	 * @param string $email        Email address.
 	 * @param string $display_name Reader display name to be used on account creation.
 	 * @param bool   $authenticate Whether to authenticate after registering. Default to true.
+	 * @param object $metadata Any metadata to pass along to the action hook.
```

Comment:
```suggestion
	 * @param array  $metadata     Any metadata to pass along to the action hook.
```

Linting and proper variable naming, `object` is [something else](https://www.php.net/manual/en/language.types.object.php)